application/src/test/java/uk/nhs/hee/tis/revalidation/JsonSerializationTest.java

```diff
@@ -36,11 +36,15 @@
 import uk.nhs.hee.tis.revalidation.controller.DoctorsForDBController;
 import uk.nhs.hee.tis.revalidation.dto.DoctorsForDbDto;
 import uk.nhs.hee.tis.revalidation.service.DoctorsForDBService;
+import uk.nhs.hee.tis.revalidation.service.RecommendationElasticSearchService;
 
 @ExtendWith(MockitoExtension.class)
 @WebMvcTest(DoctorsForDBController.class)
 class JsonSerializationTest {
 
+  @MockBean
+  private RecommendationElasticSearchService recommendationElasticSearchService;
```

Comment:
I know this PR doesn't create this issue but this test should not be a `@WebMvcTest`.  I'm assuming the use of `@WebMvcTest(DoctorsForDBController.class)` is the only reason that you've needed to Mock the new dependency here.

Reply:
I'll get this in a later PR